docs/src/components/ReactExample/Example.tsx

```diff
@@ -66,9 +66,14 @@ const Example = ({
 }: Props) => {
   const [isEditorOpened, setOpenEditor] = React.useState(false);
   const [isPlaygroundOpened, setPlaygroundOpened] = React.useState(false);
+  const [bg, setBackground] = React.useState<BgType>("white");
```

Comment:
Use a more descriptive name for `bg` to differentiate it from `background`, like `selectedBackground`.